fendermint/vm/interpreter/src/fvm/state/exec.rs

```diff
@@ -232,7 +254,7 @@ where
 
     /// Identity of the block creator, if we are indeed executing any blocks.
     pub fn validator_pubkey(&self) -> Option<PublicKey> {
```

Comment:
rename this to `block_producer`?

Reply:
Good catch!

Reply:
Addressed in https://github.com/consensus-shipyard/ipc/pull/1173/commits/5d96099fb019f60014abadeb77bc3ae72e9d48d4.